housekeeping-vacuum-tool/src/main/scripts/common.sh

```diff
@@ -33,6 +33,9 @@ else
   export HOUSEKEEPING_CLASSPATH=$HOUSEKEEPING_CLASSPATH:$HOUSEKEEPING_LIBS
 fi
 
+HOUSEKEEPING_CONF=$HOUSEKEEPING_TOOL_HOME/conf/
```

Comment:
Not sure which PR you want to add this but the change is also in the readme-update pr